src/tsdb/common.act

```diff
@@ -0,0 +1,23 @@
+class Metric(object):
+    @property
+    name: str
+    @property
+    tags: dict[str, str]
+    @property
+    value: float
+    @property
+    timestamp: int
+
+    def __init__(self, name: str, tags: dict[str, str], value: float, timestamp: int):
+        self.name = name
+        self.tags = tags
+        self.value = value
+        self.timestamp = timestamp
+
+    def __str__(self):
+        # XXX including str(self.timestamp) in this repr triggers a segfault for reasons unknown
```

Comment:
Is this still true?

Reply:
Seems not given that it's in the following line with no problems. Removed.